### PR TITLE
Feature/eureka form template fixes

### PR DESCRIPTION
**Pull request by rogerclotet-wefox**
Several fixes in the Eureka Form Tempalte.
Also added storyboard for each Scene created with the Eureka Form Tempalate.

### CODE REVIEW

File: Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@
 //  see http://clean-swift.com
 //
 
-final class ___VARIABLE_sceneName___Presenter: ___VARIABLE_sceneName___PresentationLogic {
+final class ___VARIABLE_sceneName___Presenter {
```
Comment (xramos) [inline Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift]: why do you remove PresentationLogic protocol here?

File: Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift [hunk 2]
```diff
@@ -23,7 +23,7 @@ extension ___VARIABLE_sceneName___Presenter: ___VARIABLE_sceneName___Presentatio
 
     func presentOnViewDidLoad(response: ___VARIABLE_sceneName___.OnViewDidLoad.Response) {
 
-        let viewModel = ___VARIABLE_sceneName___.OnViewDidLoad.ViewModel()
+        let viewModel = NewRegister.OnViewDidLoad.ViewModel(form: response.form)
```
Comment (xramos) [inline Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift]: why NewRegister instead of __VARIABLE_sceneName__?

Comment (rogerclotet-wefox) [reply Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift]: Fixed, this was a mistake. Thanks Xavi!

File: Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift (hunk 1 above)
Comment (rogerclotet-wefox) [reply Clean Swift WefoxGroup/Scene.xctemplate/FormViewController/___FILEBASENAME___Presenter.swift]: It has been moved that implementation in the extension in line 22.

**Pull request merged**